src/features/VisualNovel/pages/OverviewPage/OverviewPage.tsx

```diff
@@ -175,14 +175,16 @@ export const OverviewPage: FC = () => {
             {staffBlock}
           </div>
           <ContentWrapper isLoading={isCharactersLoading} error={charactersError}>
-            <div>
-              <Heading as="h3" size="sm">
-                Characters
-              </Heading>
-              <div className={cls.characters}>
-                {charactersBlock}
+            {characters != null && characters.length > 0 && (
+              <div>
+                <Heading as="h3" size="sm">
+                  Characters
+                </Heading>
+                <div className={cls.characters}>
+                  {charactersBlock}
+                </div>
```

Comment:
Let's use `Box`

Reply:
I'll revise it in https://github.com/vndbre/vndbre/issues/124